### ISSUE TITLE
Animation runs repeatedly in interactive mode with loop={false}

### ISSUE THREAD

**Issue by zackdotcomputer** [Gamote/lottie-react#71]
**Describe the bug**
When set to not loop and play the animation once in a block, the expectation I have is that upon entering the block the animation will play once and then not repeat. Instead, the result is that the animation plays once and then replays again on each incremental interaction.

**To Reproduce**
1. Set up an animation with the props:
```jsx
        loop={false}
        interactivity={{
          mode: "scroll",
          actions: [
            { visibility: [0, 0.3], type: "stop", frames: [0] },
            { visibility: [0.3, 1.0], type: "play", frames: [0, 38] }
          ]
        }}
```
2. Scroll until 30% visible and observe one playthrough
3. Scroll until 31% visible and observe a second playthrough

**Expected behavior**
The expectation would be that the playthrough would happen once while the user was in section 30%-100% and then not play again.

**Desktop (please complete the following information):**
 - OS: Mac
 - Browser: Chrome
 - Version: 105 (Latest)

**Comment by zackdotcomputer** [comment]
Relatedly, if one tries to circumvent this by taking a small section as your "play" section and then setting it as a "stop" section after the user leaves the section, it still doesn't work. If the user scrolls through the slice too quickly, it just jumps to the last frame:
```jsx
        interactivity={{
          mode: "scroll",
          actions: [
            { visibility: [0, 0.3], type: "stop", frames: [0] },
            { visibility: [0.3, 0.31], type: "play", frames: [0, 38] },
            { visibility: [0.31, 1.0], type: "stop", frames: [38] }
          ]
        }}
```

**Comment by Ourselp** [comment]
Hey @zackdotcomputer 

Did you find a solution for this at the end ?
I'm facing the same problem right now

**Comment by zackdotcomputer** [comment]
No, sadly.

**Comment by rcole1919** [comment]
Yes, I have same problem. You might add some loop of the animation end, and the problem will be solved

**Comment by Gamote** [comment]
v3 has the behaviour you expected as a first-class option. `lottieInView({ once: true })` plays the animation the first time it enters the viewport and then stops watching, so scrolling around inside the section replays nothing; `amount` sets how much has to be visible before it counts as entered:

```tsx
import { Lottie, LottieInteractions, lottieInView } from "lottie-react";

<LottieInteractions interactions={[lottieInView({ once: true, amount: 0.3 })]}>
  <Lottie src="/anim.json" />
</LottieInteractions>
```

`loop` defaults to `false` in v3, so nothing else is needed for a single playthrough. @s7aks, this is the `playOnce` you pointed at, under a different name. Docs: https://lottiereact.com/docs/interactions/in-view

Closing as done.

> [!NOTE]
> ### 🎉 lottie-react v3 is out
>
> A rewrite, and the things people asked for the most are in: `src` fetches a URL or takes the object, `loop`, `speed` and `direction` are reactive, `seek` moves by frame, marker, percent or seconds, an accessible control bar, scroll and in-view interactions, `LottieSvg` and `LottieLight` builds, server rendering with no dynamic import, React 18 and 19, only what you import ships, and more.
>
> [Docs and live examples](https://lottiereact.com) · [Migration guide](https://lottiereact.com/docs/migration), every v2 surface mapped to its v3 home.
